_locales/_untranslated_en.json

```
@@ -80,5 +80,11 @@
   },
   "contact_is_blocked": {
     "message": "This contact is blocked"
+  },
+  "multidevice_connecting": {
+    "message": "Connecting devices"
```

```suggestion
    "message": "Connecting devices..."
```
but indeed maybe it's not needed at all.